packages/core/test/lib/commands/compile.js

```diff
@@ -26,22 +26,18 @@ describe("compile", function () {
       }
     };
     config.network = "default";
-    config.logger = { log: val => val && memStream.write(val) };
+    config.logger = {log: val => val && memStream.write(val)};
   });
 
-  after("Cleanup tmp files", function (done) {
-    glob("tmp-*", (err, files) => {
-      if (err) done(err);
-      files.forEach(file => fs.removeSync(file));
-      done();
-    });
+  after("Cleanup tmp files", async () => {
```

Comment:
[Mocha says](https://mochajs.org/#arrow-functions) to use `function () {}` not `() => {}`:

```suggestion
  after("Cleanup tmp files", async function () {
```